backend/config.py

```diff
@@ -0,0 +1,32 @@
+from diart import PipelineConfig
+
+DIARIZATION_PIPELINE_CONFIG = PipelineConfig(
+    duration=5,
```

Comment:
Since I can't directly modify PipelineConfig (it's part of DIart), I've made the parameter values constant values in the config file.

Here it is:
```python
SLIDING_WINDOW_LENGTH = 5
PROBABILITY_THRESHOLD = 0.5  # Diart will recognize a speaker if the probability of them speaking is higher than this
MINIMUM_SPEECH_IMPROVEMENT_LENGTH = 0.1  # Diart will self-improve its speaker recognition for a speaker if the speaker speaks for longer than this
NEW_SPEAKER_THRESHOLD = 0.57  # Threshold for new speaker detection, the lower, the more sensitive Diart is to speech differences and will be more likely to generate new speakers

DIARIZATION_PIPELINE_CONFIG = PipelineConfig(
    duration=SLIDING_WINDOW_LENGTH,
    step=0.5,
    latency="min",
    tau_active=PROBABILITY_THRESHOLD,
    rho_update=MINIMUM_SPEECH_IMPROVEMENT_LENGTH,
    delta_new=NEW_SPEAKER_THRESHOLD
)
```